make/common/JdkNativeCompilation.gmk

```diff
@@ -266,6 +266,19 @@ define SetupJdkExecutableBody
     $1_LIBS += $(LIBCXX)
   endif
 
+  ifeq ($$($1_SRC), )
```

Comment:
General note: There amount of duplication between SetupJdkLibrary and SetupJdkExecutable is quickly becoming staggering... When I'm done with this cleanup, I'll unify them.